app/experimenter/legacy-ui/core/css/experimenter.css

```diff
@@ -244,3 +244,105 @@ pre.json {
   color: red;
   font-weight: bold;
 }
+
```

Comment:
Maybe we should just wrap these in a comment indicating we'll need to lift them into nimbus later so it's easy to copy them wholesale